integrations/terraform-cloud/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.9 (2024-01-25)
+
+### Bug Fixes
+
+- Fixed bug in setting terraform webhook (PORT-6201)
```

Comment:
Could make it more descriptive

Fixed a bug that prevented the creation of workspace webhooks especially when the notification configuration is set to enabled